src/psyclone/tests/domain/gocean/goloop_test.py

```diff
@@ -133,10 +133,10 @@ def test_goloop_bounds_invalid_iteration_space():
     # Set the iteration space to something invalid
     gojloop._iteration_space = "broken"
     with pytest.raises(GenerationError) as err:
-        gojloop._upper_bound()
+        gojloop.upper_bound()
```

Comment:
Copyright date?

Reply:
I think it is already up-to-date (maybe by a merge?)